update-index

```diff
@@ -131,6 +133,8 @@ def parse_design(f, everything_is_a_design: bool, log) -> Optional[Design]:
                     a_dev = a_dev.replace('|', '')
                     dev.append(a_dev)
 
+    f = f.replace(os.sep, '/')
```

Comment:
aside from windows support it does not break linux flow?

Reply:
I think I see why this works: this is the path that's written out to the markdown file. That always (?) uses Unix-style paths.

Reply:
This is a noop on linux since it replaces `/` with `/` but on windows it replaces `\` with `/`.  Yes it will always result in unix/web-style paths being written.